Automattic/jetpack
# Fix/rdv regressions on treatment

Fixes several issues with the remove duplicate views experiment.

## Proposed changes:

* Fix the Admin menu upsell notice
* Fix the Stats and Promote with Blaze post menu items
* Fix the Tour overlay being displayed on Blaze and Stats pages

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test on Simple and Atomic sites
* Go to WP-Admin > Posts
* Notice that the Promote with Blaze and Stats links are working (post list)
* Notice that Settings > Advertising is working (in-page WP-Admin screen)
* Notice that the upsell banner in WP-Admin menu is styled correctly
* Go to a Stats page from the posts lists
* Notice that there's no background overlay (from the tour component)
